assets/src/application/ui/calendars/dateDisplay/CalendarDateSwitcher/index.tsx

```diff
@@ -12,8 +12,10 @@ const CalendarDateSwitcher: React.FC<CalendarDateSwitcherProps> = React.memo(
 	({ className, displayDate = DisplayStartOrEndDate.start, labels, ...props }) => {
 		const startDate = parseISO(props.startDate) || PLUS_ONE_MONTH;
 		const endDate = parseISO(props.endDate) || PLUS_TWO_MONTHS;
+		const { footerPast, footerFuture, footer = '' } = labels;
+		const footerText = footerPast && footerFuture ? switchTenseForDate(endDate, footerPast, footerFuture) : footer;
```

Comment:
`labels` may be `undefined`, so it's better to set it to `{}` by default.

```suggestion
	({ className, displayDate = DisplayStartOrEndDate.start, labels = {}, ...props }) => {
		const startDate = parseISO(props.startDate) || PLUS_ONE_MONTH;
		const endDate = parseISO(props.endDate) || PLUS_TWO_MONTHS;
		const { footerPast, footerFuture, footer = '' } = labels;
		const footerText = footerPast && footerFuture ? switchTenseForDate(endDate, footerPast, footerFuture) : footer;
```